src/actions/get_texts.py

```diff
@@ -0,0 +1,8 @@
+from talon import actions
+
+
+def get_texts(targets: dict, show_decorations: bool = None):
+    """Get target texts"""
+    return actions.user.cursorless_single_target_command_get(
```

Comment:
Nice; cool to see this working.  I believe this is the first example of using command server to return something

Reply:
I believe so too and it just worked out of the box ;)

Reply:
Ha well that definitely doesn't happen every time 😅